en/lessons/gravity-model.md

```diff
@@ -0,0 +1,807 @@
+  <tr><td>Rutland</td><td>2</td><td>4</td><td>-2</td></tr>
+  <tr><td>Shropshire</td><td>75</td><td>66</td><td>9</td></tr>
+  <tr><td>Somerset</td><td>159</td><td>145</td><td>14</td></tr>
+  <tr><td>Staffordshire</td><td>82</td><td>85</td><td>-3</td></tr>
+  <tr><td>Warwickshire</td><td>104</td><td>70</td><td>34</td></tr>
+  <tr><td>Westmorland</td><td>5</td><td>5</td><td>0</td></tr>
+  <tr><td>Wiltshire</td><td>99</td><td>95</td><td>4</td></tr>
+  <tr><td>Worcestershire</td><td>94</td><td>53</td><td>41</td></tr>
+  <tr><td>Yorkshire</td><td>127</td><td>207</td><td>-80</td></tr>
+
+</table>
+
+
+
+## Step 4 - Historical Interpretation
+
+At this stage, the modelling process is complete and the final stage is historical interpretation.
+
+The original published article upon which this case study was based, is devoted primarily to interpreting what the results of the modelling mean to our understanding of lower class migration in the eighteenth century. As seen in the map in Figure 5, there were parts of the country that the model strongly suggested were either over- or under-sending lower class migrants to London.
+
+The co-authors offered their interpretations as to why those patterns may have appeared. These interpretations varied by place. In areas of the North of England that were rapidly industrialising, such as Yorkshire or Manchester, the opportunities locally appeared to give people fewer reasons to leave, resulting in lower than expected migration to London. In declining areas to the west, such as Bristol, the lure of London was stronger as more people left seeking work in the capital.
+
+Not all of the patterns were expected. Northumberland in the far north east proved to be a regional anomaly, sending far more (female) migrants to London than we would expect to see. Without the outputs of the model, it is unlikely that we would have thought to consider Northumberland at all, particularly because it was so far from the Metropolis and we presumed would have weak ties to London. The model thus provided new evidence for us to consider as historians and changed our understanding of the London-Northumberland relationship. A full discussion of our findings can be read in the original article.[^18]
+
+
+# Taking Your Knowledge Forward
+
+After having tried this example problem, you should have a clear understanding of how to use this example formula, as well as whether or not a gravity model might be an appropriate solution for your research problem. You have the experience and vocabulary to approach and discuss gravity models with an appropriately mathematically literate collaborator should you need to, who can help you to adapt it to your own case study.
+
+If you are fortunate enough to also have data about migrants moving to late eighteenth century London and you want to model it using the same five variables listed above, this formula would work as-is - there's an easy study her
```

Comment:
These journals need italics

Reply:
I don't see any journals. I've checked through and all journal names seem to be italicized.